relay/lcp.go

```diff
@@ -397,18 +397,18 @@ func (pr *Prover) updateELC(ctx context.Context, elcClientID string, includeStat
 
 	// 2. query the header from the upstream chain
 
-	headers, err := pr.originProver.SetupHeadersForUpdate(ctx, NewLCPQuerier(pr.lcpServiceClient, elcClientID), latestHeader)
+	headerStream, err := pr.originProver.SetupHeadersForUpdate(ctx, NewLCPQuerier(pr.lcpServiceClient, elcClientID), latestHeader)
 	if err != nil {
 		return nil, err
 	}
-	if len(headers) == 0 {
-		return nil, nil
-	}
 
 	// 3. send a request that contains a header from 2 to update the client in ELC
 	var responses []*elc.MsgUpdateClientResponse
-	for _, header := range headers {
-		anyHeader, err := clienttypes.PackClientMessage(header)
+	for h := range headerStream {
+		if h.Error != nil {
+			return nil, h.Error
```

Comment:
Can you add more info to the error messages, similar to `SetupHeadersForUpdate()`?

Reply:
@bluele thank you for suggestion. I fixed it: https://github.com/datachainlab/lcp-go/pull/55/commits/0070614f86a7bb0bf51a2c82ac3ef962e0639f39